xmtp_api_grpc/src/grpc_api_helper.rs

```diff
@@ -288,6 +337,56 @@ impl XmtpApiSubscription for Subscription {
     }
 }
 
+type EnvelopeStream = Pin<Box<dyn Stream<Item = Result<Envelope, Error>> + Send>>;
+
+pub struct GrpcMutableSubscription {
+    inner: Abortable<EnvelopeStream>,
+    channel: futures::channel::mpsc::UnboundedSender<SubscribeRequest>,
```

Comment:
I wonder if these could be named more clearly, for example `envelopes` and `requests`, or `output` and `input`

Reply:
That's fair. Let me update to some more thought out names